pkg/k8sInformer/constants.go

```diff
@@ -25,3 +25,7 @@ type ReleaseDto struct {
 func (r *ReleaseDto) getUniqueReleaseIdentifier() string {
 	return r.Namespace + "_" + r.Name
 }
+
+const (
+	ReleaseNotFoundCacheMissError = "release not found in cache"
```

Comment:
this will mostly be an error. we should introduce a error for this instead of const. 
```go
var ReleaseNotFoundCacheMissError = errors.New(ReleaseNotFoundCacheMissError)
```
Also we should define a util to check if error is ReleaseNotFoundCacheMissError